.github/workflows/close-stale-prs.yml

```diff
@@ -0,0 +1,20 @@
+name: Close stale PRs
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4
+        with:
+          days-before-close: 0
+          days-before-stale: 14
+          # Temporary, avoid closing PRs until this action merged
+          debug-only: true
+          exempt-pr-labels: RFC,Waiting on Review
```

Comment:
Might make more sense to instead require `Waiting on Author`?

Reply:
We're not using that label too often but I'm open to switching. I'll have to go through old PRs and label them accordingly but that's not a big problem.

Reply:
Maybe I misunderstand, but I think that `Waiting on Author` should be closed after some time if the author does not respond. Not really sure about `Waiting on Review` – keeping open forever might not make sense, but on the other hand, automatically closing after some time without even a minimal review appears to be somewhat rude.

Reply:
I think Nikita is suggesting to only close issues that have been specifically marked with `Waiting on Author`. The current implementation would close those, but also all other issues that aren't labeled with `RFC` or `Waiting on Review`.

> but on the other hand, automatically closing after some time without even a minimal review appears to be somewhat rude.

Those would currently not be closed. I also think it would be somewhat rude to close them if nobody has given feedback on the PR. Of course keeping them open forever has no purpose but they should get reviewed eventually.

Reply:
I'm ok with that approach. We can mark reviewed PRs that need changes with `Waiting on Author`.